alexandria/core/apps.py

```diff
@@ -1,6 +1,17 @@
 from django.apps import AppConfig
+from django.conf import settings
+from django.utils.module_loading import import_string
 
 
 class DefaultConfig(AppConfig):
     name = "alexandria.core"
     label = "alexandria_core"
+
+    def ready(self):
+        from .models import VisibilityMixin
+
+        # to avoid recursive import error, load extension classes
```

Comment:
Comment should be before the import above

Reply:
Changed. Beware: This was an exact copy/paste from caluma and emeis.